vita3k/gui/src/main_menubar.cpp

```diff
@@ -26,10 +26,12 @@
 namespace gui {
 
 static void draw_file_menu(GuiState &gui, EmuEnvState &emuenv) {
-    auto lang = gui.lang.main_menubar.file;
+    auto &lang = gui.lang.main_menubar.file;
     if (ImGui::BeginMenu(lang["title"].c_str())) {
         if (ImGui::MenuItem(lang["open_pref_path"].c_str()))
             open_path(emuenv.pref_path.string());
+        if (ImGui::MenuItem(lang["open_texture_folder"].c_str()))
```

Comment:
Please add a check to see if the folder exists

Reply:
i think to add texture dir in app init.cpp? but i do not want add in this pr

Reply:
I'm not sure what you mean, but if the texture folder doesn't exist, an error will occur.

Reply:
Okay, let's put it another way. In master, only enable export texture to running games create this dir. Mr. @Macdu could you check this, wanna add texture dir in app init